scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -134,6 +139,11 @@ databaseChangeLog:
                   type: integer
                   constraints:
                     nullable: true
+              - column:
+                  name: guid
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
```

Comment:
@pradyutsarma  Did as you suggested in my previous pr "add guid to api server". When Michael merged the pr, he did what I did now. So follow his pattern here.